app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/LeanbackOverlayFragment.java

```diff
@@ -18,6 +20,7 @@ public class LeanbackOverlayFragment extends PlaybackSupportFragment {
     private VideoPlayerAdapter playerAdapter;
     private boolean shouldShowOverlay = true;
     private Lazy<PlaybackControllerContainer> playbackControllerContainer = inject(PlaybackControllerContainer.class);
+    private final Integer skipForwardLength = KoinJavaComponent.<UserSettingPreferences>get(UserSettingPreferences.class).get(UserSettingPreferences.Companion.getSkipForwardLength());
```

Comment:
Move the reading of the setting to the actual usage and just inject the class here.

Reply:
I have created an extra variable for the skipForwardLength. Or should the whole UserSettingPreferences be passed to the CustomSeekProvider?